icu4c/source/i18n/unicode/uformattedvalue.h

```diff
@@ -120,7 +118,7 @@ typedef struct UConstrainedFieldPosition UConstrainedFieldPosition;
  *
  * @param ec Set if an error occurs.
  * @return The new object, or NULL if an error occurs.
- * @draft ICU 64
+ * @stable ICU 64
  */
 U_DRAFT UConstrainedFieldPosition* U_EXPORT2
```

Comment:
nit: As you noted on the other PR: U_DRAFT = U_CAPI. :)
Though perhaps U_STABLE for documentation purposes?

Reply:
Oh, good catch, thanks, I am fixing several instances of this now in uformattedvalue.h, unumberformatter.h, ureldatefmt.h